src/components/Tabs.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tab: Tab,
+  selectedTabId: string,
+};
+
+export const TabLink:React.FC<Props> = ({ tab, selectedTabId }) => {
+  const {
+    id, title,
+  } = tab;
```

Comment:
Can we make it in one string?
```suggestion
  const { id, title } = tab;
```

Reply:
Sure thing! Thanks!